GeosUtil/pressure_mod.F90

```diff
@@ -517,39 +517,47 @@ SUBROUTINE INIT_PRESSURE( Input_Opt, State_Grid, RC )
     RC      = GC_SUCCESS
     ThisLoc = ' -> at Init_Pressure (in GeosUtil/pressure_mod.F90)'
 
-    ALLOCATE( PFLT_DRY( State_Grid%NX, State_Grid%NY ), STAT=RC )
+    IF (.NOT. ALLOCATED( PFLT_DRY ))
```

Comment:
Thanks @jwallwork23 for this.  I think we need to place everything within an IF / THEN block to prevent the arrays from being zeroed if INIT_PRESSURE is called twice.  Thus:
```F90
IF (.NOT. ALLOCATED( PFLT_DRY )) THEN
    ALLOCATE( PFLT_DRY( State_Grid%NX, State_Grid%NY ), STAT=RC )
    CALL GC_CheckVar( 'vdiff_mod.F90:PFLT_DRY', 2, RC )
    IF ( RC /= GC_SUCCESS ) RETURN
    PFLT_DRY = 0e+0_fp
ENDIF
```
```

Reply:
Ah yes, good point. Addressed in a02905c405361e9063381561045cc5524417a492.